src/main/java/Chandler.java

```diff
@@ -0,0 +1,113 @@
+
+                if (words.length <= 1 && !inputCommand.equals("list") && !inputCommand.equals("bye") && !inputCommand.equals("mark") && !inputCommand.equals("unmark")) {
+                    throw new ChandlerException("You need to specify a task.");
+                }
+
+                switch (inputCommand) {
+                    case "list":
+                        System.out.println(OUTPUT_INDENTATION + "Here are the tasks in your list:");
+                        for (int index = 0; index < listSize-1; index++) {
+                            System.out.println(OUTPUT_INDENTATION + (index+1) + "." + list.get(index));
+                        }
+                        break;
+                    case "mark":
+                        int task_number = Integer.parseInt(input.replace("mark ", "")) - 1;
+                        if (task_number < 0 || task_number >= list.size()) {
+                            throw new ChandlerException("Invalid task number");
+                        }
+                        System.out.println(OUTPUT_INDENTATION + "Nice! I've marked this task as done:");
+                        list.get(task_number).markAsDone();
+                        System.out.println(OUTPUT_INDENTATION + list.get(task_number));
+                        break;
+                    case "unmark":
+                        int task_number2 = Integer.parseInt(input.replace("unmark ", "")) - 1;
+                        if (task_number2 < 0 || task_number2 >= list.size()) {
+                            throw new ChandlerException("Invalid task number");
+                        }
+                        System.out.println(OUTPUT_INDENTATION + "Ok, I've marked this task as not done yet:");
+                        list.get(task_number2).markAsUndone();
+                        System.out.println(OUTPUT_INDENTATION + list.get(task_number2));
+                        break;
+                    case "todo":
+                        Todo taskTodo = new Todo(input.replace("todo ", ""), "T");
+                        list.add(taskTodo);
+                        System.out.println(
+                                OUTPUT_INDENTATION + "Awesome! Something to do without deadline hehe\n" +
+                                        OUTPUT_INDENTATION + "  " + taskTodo + "\n" +
+                                        OUTPUT_INDENTATION + "You better not procrastinate... or maybe you should");
+                        listSize++;
+                        break;
+                    case "deadline":
+                        input = input.replace("deadline ", "");
+                        int indexBy = input.indexOf("/by");
+                        String description = input.substring(0, indexBy);
+                        String by = input.substring(indexBy + 4);
+                        Deadline taskDeadline = new Deadline(description, by, "D");
+                        list.add(taskDeadline);
+                        System.out.println(
```

Comment:
Good work with keeping each line of code within the character limits prescribed for each line.